cfme/tests/cli/test_appliance_console.py

```diff
@@ -52,6 +53,31 @@
 ]
 
 
+def is_port_open(hostname, port):
+    s = socket.socket(socket.AF_INET, socket.SOCK_STREAM)
+    try:
+        s.connect((hostname, int(port)))
```

Comment:
Q: Should we close the connection?

Reply:
A: in case anyone is reading - the `finally` block will execute before the return occurs, but yes, connection has to be closed.